.bumpversion.cfg

```diff
@@ -35,3 +35,5 @@ values =
 [bumpversion:glob:model_hub/examples/mmdetection/*.yaml]
 
 [bumpversion:glob:model_hub/examples/mmdetection/hydra/configs/config.yaml]
+
+[bumpversion:file:docs/_static/version-switcher/versions.json]
```

Comment:
I'm not sure that's what we want here, I think we should select versions with major changes and update accordingly, otherwise the dropdown list would get very long. After all, users can change the url to get which ever version docs they want.

Reply:
Users could already edit the URLs before, so if you're counting on that, then we don't need a dropdown at all. The dropdown is for discoverability and to help people who don't know how to edit the URL.

If you really want to keep the length down, I think we're consistent enough with doing minor version bumps for big changes that we could stick with a policy of including exactly the last patch version of each minor version series. I think any other choice would be confusing.

Reply:
Sounds good to me, we can by default include minor version docs